.gitignore

```
@@ -1,2 +1,3 @@
 .vscode
-node_modules
\ No newline at end of file
+node_modules
+dist
```

```suggestion
custom-apps/*/dist
```
The dist folder should be kept for the extensions